_includes/v21.2/misc/s3-compatible-warning.md

```diff
@@ -0,0 +1,3 @@
+{{site.data.alerts.callout_danger}}
+Unlike Amazon S3, Google Cloud Storage, and Azure Storage options, Cockroach Labs does **not** actively test [S3-compatible services](use-cloud-storage-for-bulk-operations.html) (e.g., [MinIO](https://min.io/), [Red Hat Ceph](https://docs.ceph.com/en/pacific/radosgw/s3/)).
```

Comment:
Nit: "Unlike A, B, and C, D is a different letter." But here D is not "Cockroach Labs", but "S3-compatible services". Switching from `unlike` to `while` helps here:

> While Cockroach Labs actively tests Amazon S3, Google Cloud Storage, and Azure Storage options, we **do not** test third-party [S3-compatible services](use-cloud-storage-for-bulk-operations.html) such as [MinIO](https://min.io/) and [Red Hat Ceph](https://docs.ceph.com/en/pacific/radosgw/s3/).